### PR TITLE
Scalpel-0.5.4-GCCcore-8.3.0.eb for INC1006647

### PR DESCRIPTION
**Pull request by sihart25**
For INC1006647

`Scalpel-0.5.4-GCCcore-8.3.0.eb`
* [x] Assigned to reviewer
* [ ] EL7-cascadelake
* [ ] EL7-haswell
* [ ] EL7-sandybridge
* [ ] Ubuntu16 VM

### CODE REVIEW

File: easybuild/easyconfigs/s/Scalpel/Scalpel-0.5.4-GCCcore-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+dependencies = [
+    ('Perl', '5.30.0'),
+]
+
+files_to_copy = ['samtools-1.1', 'Microassembler', 'bamtools-2.3.0', 'bcftools-1.1', 'MLDBM', 'Text', 'Parallel',
+                 'FindSomatic.pl', 'HashesIO.pm', 'MLDBM.pm', 'scalpel-export', 'Utils.pm', 'FindDenovos.pl',
+                 'FindVariants.pl', 'LICENSE', 'MANUAL', 'README', 'scalpel-discovery', 'SequenceIO.pm',
+                 'Usage.pm', ]
+
+fix_perl_shebang_for = ['%%(namelower)s%%(version)s/%s' % x for x in ['FindDenovos.pl', 'FindVariants.pl',
```
Comment (orbsmiv) [inline easybuild/easyconfigs/s/Scalpel/Scalpel-0.5.4-GCCcore-8.3.0.eb]: What is this bit for?!

Comment (sihart25) [reply easybuild/easyconfigs/s/Scalpel/Scalpel-0.5.4-GCCcore-8.3.0.eb]: List of files for which Perl shebang should be fixed to ‘#!/usr/bin/env perl’ (glob patterns supported)

Comment (sihart25) [reply easybuild/easyconfigs/s/Scalpel/Scalpel-0.5.4-GCCcore-8.3.0.eb]: as used in BATVI easybuild/easyconfigs/b/BatVI/BatVI-1.02-foss-2019a.eb

Comment (sihart25) [reply easybuild/easyconfigs/s/Scalpel/Scalpel-0.5.4-GCCcore-8.3.0.eb]: SimonB told me to do it!

**Comment by orbsmiv** [comment]
built on all requested platforms

**Pull request merged**